<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryGeometryScatterPlotsWidget</class>
 <widget class="QWidget" name="GeometryGeometryScatterPlotsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="settingsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="4" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Save Options</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="save_btn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string> Save Scatter Plots</string>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/icons/img/image_icon.png</normaloff>:/icons/img/image_icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Show Options</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="x_axis_label">
              <property name="text">
               <string>X Axis</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="x_axis_cb"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="y_axis_label">
              <property name="text">
               <string>Y Axis</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="y_axis_cb"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Scatter Plot Options</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="0" colspan="2">
             <widget class="QWidget" name="marker_settings_widget" native="true"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="marker_size_dsb"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Marker Size</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
